lib/workers/bin/worker.rb

```diff
@@ -1,2 +1,10 @@
 type = ARGV.shift
+begin
+  # Remove the ::Runner class which is in all of them.
+  worker_name = type.gsub(/::Runner$/, '')
+  # Will set the process' name to eg. 'EVM Worker: MiqPriorityWorker'
+  # in ps aux, htop, ...
+  Process.setproctitle("EVM Worker: #{worker_name}")
```

Comment:
What about the `ManageIQ::Providers::blahblah::EventCatcher`? If you want to shorten the names to minimum, this one is probably going to stay as it is, right?